com.unity.netcode.gameobjects/Runtime/Interest/InterestManager.cs

```diff
@@ -3,10 +3,19 @@
 namespace Unity.Netcode.Interest
 {
     // interest *system* instead of interest node ?
-    internal class InterestManager<TObject>
+    public class InterestManager<TObject>
     {
         private InterestNodeStatic<TObject> m_DefaultInterestNode = new InterestNodeStatic<TObject>();
 
+        public bool Disable;
```

Comment:
For single bools like this, I usually try to avoid double negatives.
In our case here, we'd have Disable = false to enable it... just a clarity thing

Reply:
+1 for inverting logic to be `bool Enable = true`

Reply:
I see your point and generally agree...

But I felt like if I was a dev and saw that it was `Enable`, I would assume it was by disabled by default, which is the opposite way I want them to think about it 🤷🏽‍♂️